code-studio/src/js/react-only.js

```diff
@@ -6,3 +6,4 @@
 
 window.React = require('react');
 window.ReactDOM = require('react-dom');
+window.Radium = require('radium');
```

Comment:
react-only is loaded on embed_blocks pages (like https://studio.code.org/s/allthethings/stage/13/puzzle/3). The name is becoming slightly less accurate with the addition of Radium, but Radium is still pretty React related so I don't feel too badly.